src/__benchmarks__/GraphQLSchema.bench.ts

```diff
@@ -0,0 +1,16 @@
+import { bench, describe } from 'vitest';
+
+import { GraphQLSchema } from '../type/schema.js';
+
+import { buildClientSchema } from '../utilities/buildClientSchema.js';
```

Comment:
Generally it might be preferred to use this from the built output as we use a lot of utils to transpile which means that the built output will have different performance characteristics from this output.